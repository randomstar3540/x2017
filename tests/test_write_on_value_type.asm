FUNC LABEL 0
    MOV VAL 3 REG 0
    RET
