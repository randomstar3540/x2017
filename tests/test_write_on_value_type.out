ERROR: Writing on value type!
